src/wallet/rpc/backup.cpp

```diff
@@ -1772,6 +1772,7 @@ RPCHelpMan listdescriptors()
                         {RPCResult::Type::NUM, "", "Range end inclusive"},
                     }},
                     {RPCResult::Type::NUM, "next", /*optional=*/true, "The next index to generate addresses from; defined only for ranged descriptors"},
```

Comment:
If this is kept for compatibility, it might be good to mention it, and that the value is equal to the other. Otherwise users might be confused to see the same field twice and wonder what the difference is?

Reply:
Done in https://github.com/bitcoin/bitcoin/commit/b082f28101773e0ef0281d97025e65d0364d6f29.